### ISSUE TITLE
Where does flag configuration code go in a Rails project? (can't get Intl to work when pre-rendering a react component)

### ISSUE THREAD

**Issue by gregblass** [rubyjs/mini_racer#150]
Is there a specific place I should be placing this in a Rails project?

`MiniRacer::Platform.set_flags! :harmony`

I have it in config/initializers/mini_racer.rb

However, I'm still getting this error when trying to use `Intl` when prerendering a component:

`React::ServerRendering::PrerenderError (Encountered error "#<ExecJS::ProgramError: ReferenceError: Intl is not defined>" when prerendering`

Or, could it be possible that Intl just still not supported, even with :harmony enabled?

**Comment by nightpool** [comment]
It looks like Intl support has to be enabled independently: https://v8.dev/docs/i18n#embedding-v8

**Comment by nightpool** [comment]
Ah, but see here wrt i18n support: https://github.com/cowboyd/libv8/issues/278

**Comment by ksweetie** [comment]
I've got the same error trying to do the same thing. Did you ever figure this out?

**Comment by nightpool** [comment]
This is fixed by #166 

```ruby
c = MiniRacer::Context.new

c.eval("['1','b','c','a','\u00E4'].sort(new Intl.Collator('en').compare);")
# => ["1", "a", "\u00E4", "b", "c"]

c.eval("['1','b','c','a','\u00E4'].sort(new Intl.Collator('sv').compare);")
# => ["1", "a", "b", "c", "\u00E4"]
```

**Comment by tisba** [comment]
Has your initial problem been solved, @ksweetie?
